src/bin/cli/mod.rs

```diff
@@ -682,3 +697,41 @@ fn explicit_args(matches: &ArgMatches) -> Vec<&Id> {
         })
         .collect()
 }
+
+/// A helper function to check if the target name provided by the user matches the target name in solang.toml
+/// Returns false if the target names do not match
+/// Returns true if the target names match or if solang.toml does not exist in the current directory
+fn check_target_match(target_name: &str) -> bool {
+    // Get the manifest path from the current directory
+    let manifest_path = PathBuf::from("solang.toml");
```

Comment:
The configuration file name and location isn't static; it can be overwritten via `--config-file`